src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -161,13 +161,6 @@ private static ProcessModuleCollection GetModules(int processId, bool firstModul
                             continue;
                         }
 
-                        var module = new ProcessModule()
```

Comment:
nice refactor! 👍

Reply:
It was either that or do weird things will null suppressions :)